ui/src/app/pages/data/concept/concept-homepage.tsx

```diff
@@ -262,25 +241,24 @@ export const ConceptHomepage = withCurrentWorkspace()(
           domain: domain.domain,
           items: []
         }));
+        // Add ConceptCacheItem for Surveys tab
+        conceptsCache.push({domain: Domain.SURVEY, items: []});
         let conceptDomainCounts: DomainCount[] = conceptDomainInfo.items.map((domain) => ({
           domain: domain.domain,
           name: domain.name,
           conceptCount: 0
         }));
-        if (environment.enableNewConceptTabs) {
-          conceptsCache = [...conceptsCache, ...conceptCacheStub];
-          conceptDomainCounts = [...conceptDomainCounts];
-        } else {
+        if (!environment.enableNewConceptTabs) {
```

Comment:
@dolbeew we should probably only shot this in local for the time being.

Reply:
Yes, currently this flag is only enabled in the local environment